packages/frontend/components/KeyList.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <div v-for="key in keys">
+     <a :href="`/provenance/${key}`">{{key}}</a>
+    </div>
+</template>
+
+<script>
+import { myDate } from '~/utils/dateUtility';
```

Comment:
I don't see where myDate is being used. Also look into using the built in object Date(). If you pass in a timestamp, it will output a readable string.